buildpack/start.py

```diff
@@ -163,6 +167,7 @@ def _register_signal_handlers():
             extra_jmx_instance_config=databroker_jmx_instance_cfg,
             jmx_config_files=databroker_jmx_config_files,
         )
+        fluentbit.update_config(m2ee)
```

Comment:
I notice `fluenbit.update_config()`is called twice ?

Reply:
Yes, it is mistake, didn't notice it, changed